bigbang/bigbang_io.py

```diff
@@ -164,9 +164,9 @@ def to_mbox(msgs: list, dir_out: str, filename: str):
         mbox.unlock()
 
 
-class ListservArchiveIO:
+class ArchiveIO:
     """
-    This class handles the data transformations for Listserv Archives.
+    This class handles the data transformations for Archives.
```

Comment:
These comments might be a good place to elucidate the difference between a List and an Archive.

I'll admit I'm fuzzy on this at the moment.

I believe Archives can sometimes be composed of multiple Lists. Is that right?

Reply:
I added a pointer to more detailed descriptions on the differences between lists and archives in the docstrings in this PR.